### ISSUE TITLE
Error: missing field `analyzeDependencies`

### ISSUE THREAD

**Issue by lookfirst** [parcel-bundler/lightningcss#290]
parcel --version
2.7.0

I get the feeling that this is related to this commit: [676bf1fefc46a75d7098efbc53e744d89d47ba93](https://github.com/parcel-bundler/lightningcss/commit/676bf1fefc46a75d7098efbc53e744d89d47ba93) by @devongovett 

```
parcel build index.html
🚨 Build failed.

@parcel/optimizer-css: missing field `analyzeDependencies`

  Error: missing field `analyzeDependencies`
      at Object.optimize (node_modules/@parcel/optimizer-css/lib/CSSOptimizer.js:217:57)
      at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
      at async PackagerRunner.optimize (node_modules/@parcel/core/lib/PackagerRunner.js:364:20)
      at async PackagerRunner.getBundleResult (node_modules/@parcel/core/lib/PackagerRunner.js:255:15)
      at async getInlineBundleContents (node_modules/@parcel/core/lib/PackagerRunner.js:310:21)
      at async $bc66accb63b05e9a$export$f074a8f9bef628fd (node_modules/@parcel/utils/lib/index.js:35318:30)
      at async PackagerRunner.package (node_modules/@parcel/core/lib/PackagerRunner.js:294:14)
      at async PackagerRunner.getBundleResult (node_modules/@parcel/core/lib/PackagerRunner.js:253:20)
      at async PackagerRunner.getBundleInfo (node_modules/@parcel/core/lib/PackagerRunner.js:239:9)
      at async PackagerRunner.run (node_modules/@parcel/core/lib/PackagerRunner.js:160:183)
```

**Comment by lookfirst** [comment]
Ok, more experimentation, falling back to parcel 2.6.2 is also failing. Going to keep going back until it works again. I had upgraded from 2.4.1.

**Comment by lookfirst** [comment]
Ok, even stranger... 2.4.1 is failing now too.

**Comment by lookfirst** [comment]
I gave up and just did --no-optimize for now. This is really weird, if you want to  ͡° ͜ʖ ͡° into it, the project is painfully simple:

https://github.com/lookfirst/mui-rff/tree/master/example

**Comment by manu-optus** [comment]
You can temporarily revert to the older version of lightningcss until the issue is resolved. This seems to have originated from version 1.15.0.

If you are using `yarn`, then adding resolutions to your package.json and installing dependencies should help. 

1. Add this to your package.json
```
  "resolutions": {
        "lightningcss": "1.14.0"
    },
```

2. Install dependencies
 `yarn install`

**Comment by lookfirst** [comment]
Thank you. Confirmed that that resolves this issue.

**Comment by devongovett** [comment]
Sorry about that! Published a fix in v1.15.1.

**Comment by lookfirst** [comment]
@devongovett No worries, thanks for the fast fix. I just wonder why you don't have some unit tests that would have caught that.
